app/services/rbac.rb

```diff
@@ -35,6 +39,21 @@ def load_user_permissions(identity)
       end
     end
 
+    # Scoped for inventory groups
+    def definition_value(definition, selector)
+      if definition[:operation] == 'in' &&
+         definition[:key] == "#{selector.keys.first.to_s.pluralize}.#{selector.values.first}" &&
+         definition[:value].instance_of?(Array)
+        # Received '[nil]' symbolizes access to ungrouped entries.
+        # In output represtented with an empty array.
+        definition[:value].map { |dv| dv || [] }
+      elsif definition[:operation] == 'equal' &&
+            definition[:key] == "#{selector.keys.first}.#{selector.values.first}" &&
+            definition[:value].instance_of?(String)
+        definition[:value]
+      end
```

Comment:
There might be other uncovered outputs that would result in `nil` which make the edge-case described above uncatchable. Please make sure if we REALLY need to support the `equal` syntax.

Reply:
We don't.